src/mongo/delegates/SearchIndexer.class.php

```diff
@@ -234,7 +224,11 @@ public function generateSearchDocuments(
                     array($searchDocumentType)
                 );
 
-                $this->getApplyOperation()->createJob(array($subject), $queueName, $jobOptions);
+                $subjects[] = $subject;
```

Comment:
Is this block worth a comment to explain what it's doing now that it's not as simple as 1 create job for every item in the for loop?